app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -31,6 +31,8 @@
   heading_level ||= nil
   prefix ||= nil
   suffix ||= nil
+  text_direction_rtl ||= false
+  text_direction_label_ltr ||= false
```

Comment:
It might just be me, but reading this with one option referring to `rtl` and the other being `ltr` is a bit confusing. Is there a reason they're not both `_rtl` or both `_ltr`?